utils/gitlog2changelog.py

```diff
@@ -131,4 +137,4 @@
 
 # Close the input and output lines now that we are finished.
 fin.close()
-fout.close()
+fout.close()
```

Comment:
```suggestion
fout.close()

```


The only change here was removing the final newline.

In Linux world, files should have a final newline by convention. It also helps when printing multiple files one after the other with `cat`.

Reply:
thanks for the notice, the new line was added.

Reply:
> In Linux world, files should have a final newline by convention.

Of potential interest: I looked into it a bit more in the past because of different CSV dialects and while this is usually represented as something along the lines of "final newline by convention", another, and potentially a better, specification is that Windows uses CRLF and it is a line separator, so it is in between the lines, while Linux uses LF and it is line ending, so it is after each line.

Reply:
So, what does that mean? What do we need to choose?

Reply:
Sorry, I didn't mean to bring confusion. The files are to be committed (stored within Git) in Linux style, so with LF as _line ending_ (not CRLF as _line separator_ or any combination of the two styles). GitHub complains are justified, LF should be the last character in a file.